testing/mock/ibc_app.go

```diff
@@ -88,8 +88,8 @@ type MockIBCApp struct {
 }
 
 // NewMockIBCApp returns a MockIBCApp. An empty PortID indicates the mock app doesn't bind/claim ports.
```

Comment:
```suggestion
// NewIBCApp returns a IBCApp. An empty PortID indicates the mock app doesn't bind/claim ports.
```

Reply:
I think this suggestion from @colin-axner is still valid.